src/test/components/ProfileCallTreeView.test.js

```diff
@@ -65,7 +65,7 @@ describe('calltree/ProfileCallTreeView', function() {
       profile = getProfileFromTextSamples(`
         A  A  A
         B  B  B
-        C  C  H
+        C  C  H[lib:libH.so]
```

Comment:
I decided to add libraries in some samples so that we could see they're _not_ dimmed.
In the snapshots, you'll see that the ones with an added "library" information won't be dimmed.